server/kadalu_quotad/quotad.py

```diff
@@ -6,10 +6,14 @@
 import json
 import logging
 
+# How do we handle this library dependency while packaging ?
```

Comment:
@aravindavk @sac need suggestion on how do we handle this issue with pypi package.

Reply:
For now, we can copy. I will think about it.

Reply:
I am working on the Makefile changes to create pypi package and push to pypi repo. Let me see how to package without copy